packages/template-retail-react-app/app/pages/checkout/index.test.js

```diff
@@ -209,17 +169,25 @@ test.skip('Can proceed through checkout steps as guest', async () => {
         // mock place order
         rest.post('*/orders', (req, res, ctx) => {
             currentBasket = {
-                ...ocapiOrderResponse,
-                customer_info: {...ocapiOrderResponse.customer_info, email: 'test@test.com'}
+                ...scapiOrderResponse,
+                customerInfo: {...scapiOrderResponse.customerInfo, email: 'test@test.com'}
             }
             return res(ctx.json(currentBasket))
+        }),
+
+        rest.get('*/baskets', (req, res, ctx) => {
+            const baskets = {
+                baskets: [currentBasket],
+                total: 1
+            }
+            return res(ctx.json(baskets))
         })
     )
 
     // Set the initial browser router path and render our component tree.
     window.history.pushState({}, 'Checkout', createPathWithDefaults('/checkout'))
     renderWithProviders(<WrappedCheckout history={history} />, {
-        wrapperProps: {siteAlias: 'uk', appConfig: mockConfig.app}
+        wrapperProps: {isGuest: true, siteAlias: 'uk', appConfig: mockConfig.app}
```

Comment:
I realized that by default, our test providers would emulate _registered_ users. Why is that btw?

How would people not familiar with that setup discover it? I think it's more logical to test _guest_ users by default, as that is aligned with the shopping experience.

Reply:
@vmarta I did not set up the original mock token, if I were to guess, having a registered token to begin with allows us to easily mock most of the pages from pdp plp , to account pages without having to initialize login.